birdhouse/components/raven/docker-compose-extra.yml

```diff
@@ -9,7 +9,7 @@ x-logging:
 
 services:
   raven:
-    image: pavics/raven:${RAVEN_VERSION}
```

Comment:
I like that we're now being consistent with how we specify images instead of before where we had some hardcoded image names and some in variables from default.env. I like that this PR will make this more consistent